lxd/storage/backend_lxd.go

```diff
@@ -1212,7 +1219,12 @@ func (b *lxdBackend) GetInstanceUsage(inst instance.Instance) (int64, error) {
 	defer logger.Debug("GetInstanceUsage finished")
 
 	if inst.Type() == instancetype.Container {
-		return b.driver.GetVolumeUsage(drivers.VolumeTypeContainer, inst.Name())
+		// There's no need to pass config as it's not needed when retrieving
+		// the volume usage.
+		vol := b.newVolume(drivers.VolumeTypeContainer, InstanceContentType(inst),
```

Comment:
@monstermunchkin please dont overflow to next line here.

Reply:
@monstermunchkin please can you use `volType, err := InstanceTypeToVolumeType(inst.Type())` rather than hardcoding the drivers.VolumeTypeContainer here (I realise this was my code that was already here, but have since tried to avoid hard coding instance type where possible), and seeing as you have instance available you can just use its Type() function to get the volume type.